.swiftlint.yml

```diff
@@ -58,10 +58,11 @@ opt_in_rules:
   - trailing_newline
   - trailing_semicolon
   - trailing_whitespace
-  - unowned_variable_capture
   - closure_spacing
   # Rules to enable after initial cleanup.
   # - weak_delegate
+  # - discouraged_object_literal
+  # - unowned_variable_capture
```

Comment:
Should be a safe change to convert to weak, but since this is supposed to be a format change, and I don't want to make any potential functionality changes